.github/workflows/ci.yml

```diff
@@ -46,3 +46,27 @@ jobs:
         run: |
           sbt test
         shell: bash
+  native-image:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [macOS-latest, ubuntu-latest]
+        include:
+          - os: macOS-latest
+            artifact: fastpass-x86_64-apple-darwin
+          - os: ubuntu-latest
+            artifact: fastpass-x86_64-pc-linux
+    steps:
+      - uses: actions/checkout@v2
+      - uses: olafurpg/setup-scala@v6
+      - uses: olafurpg/setup-scala@v6
+        with:
+          java-version: graalvm@20.1.0
+      - run: gu install native-image
+      - run: sbt native-image
+      - run: cp fastpass/target/graalvm-native-image/fastpass fastpass
```

Comment:
Actually it'll copy the native-image to `fastpass/fastpass`, and try to upload the whole directory

Reply:
Good point! 😨  Will fix